includes/fields/class-field-post-taxonomy.php

```diff
@@ -215,25 +215,44 @@ public function tax_ajax( $post_id = null ) {
     }
 
     public function tax_select( $post_id = null ) {
-        $attr = $this->field_settings;
-
+        $attr     = $this->field_settings;
         $selected = $this->terms ? $this->terms[0] : '';
-
         $required = sprintf( 'data-required="%s" data-type="select"', $attr['required'] );
 
+        $parents      = $this->field_settings['exclude'];
+        $child_ids     = [];
+        $exclude_type = $this->exclude_type;
+
+        if ( !empty( $parents ) ) {
+            foreach ( $parents as $parent ) {
+                $terms = get_terms( 'category', array(
```

Comment:
@hasinur1997 We have used both array() and [] in our plugin. This is bad and we should fix it. But for now, if you add new code, for a single php file figure out which notation we're using, is it `array()` or `[]`. Then use that notation in everywhere in that file. For this case, we're using `[]` notation in this file, so we should use `[]` instead of `array()`.